local_server/main.py

```diff
@@ -0,0 +1,25 @@
+from fastapi import FastAPI, Query
```

Comment:
I suggest we remove the entire `/local_server` layer, it's not really necessary, `/app` can be the first layer of the server

Reply:
Ok, got it